app/screens/create_direct_message/user_list.tsx

```diff
@@ -247,8 +223,7 @@ export default function UserList({
                 {...keyboardDismissProp}
                 keyExtractor={keyExtractor}
                 initialNumToRender={INITIAL_BATCH_TO_RENDER}
-                ListEmptyComponent={renderNoResults}
-                ListFooterComponent={renderLoading}
```

Comment:
If I remember correctly, the FooterComponent was added also to show the loading at the end of the list if we are still loading elements (easily visible if you add some throttling to the client (`await new Promise((resolve) => setTimeout(resolve, 1000))`) and a small page size (1)). Is this still handled?